dask/dataframe/io/tests/test_csv.py

```diff
@@ -222,6 +222,17 @@ def test_enforce_columns(reader, blocks):
         dask.compute(*dfs, get=get_sync)
 
 
+def test_robust_column_mismatch():
```

Comment:
Slight nitpick - the test file is organized by functionality, might want to move this test down to below https://github.com/dask/dask/pull/1740/files#diff-62d83a5db4152149463203311f3d8b00R269?